src/contexts/SwapContext.tsx

```diff
@@ -170,13 +182,29 @@ export const SwapContextProvider: FC<{
     () => ({ ...initialSwapContext.formProps, ...originalFormProps }),
     [originalFormProps],
   );
+  const [userSlippage, setUserSlippage] = useLocalStorage<number | undefined>('jupiter-terminal-slippage', undefined);
+  const [form, setForm] = useState<IForm>(
+    (() => {
+      const slippageBps = (() => {
+        if (props.useUserSlippage && typeof userSlippage !== 'undefined') {
```

Comment:
`typeof userSlippage !== 'undefined'` seems to be redundant

Reply:
userSlippage can be undefined if the user have not changed their slippage before, so it's required